cls/SourceControl/Git/Extension.cls

```diff
@@ -49,15 +50,20 @@ Method UserAction(Type As %Integer, Name As %String, InternalName As %String, Se
         set Name = "%SourceMenu,Status"
     } 
 
+    if Type = 1, Name = 1 {
```

Comment:
"," is not modern ObjectScript syntax - please change to:
if (Type = 1) && ((Name = 1) || (Name = 7))

Reply:
(and feel free to fix this anywhere you find it in old code!)